test-network-nano-bash/README.md

```diff
@@ -0,0 +1,83 @@
+- Minimal set of dependencies
+- Minimal requirements on Fabric version (any v2.x orderer and peer nodes are supported)
+- Minimal set of environment variable overrides of the default orderer orderer.yaml and peer core.yaml configurations
+- Minimal scripting with minimal set of reference commands to get a Fabric network up and running
+- Minimal channel configuration for an orderer organization (3 ordering nodes) and two peer organizations (with two peers each)
+- Minimal endorsement policy to allow a single organization to approve and commit a chaincode (unlike Test Network which requires both organizations to endorse)
+
+# Prereqs
+
+- Follow the Fabric documentation for the [Prereqs](https://hyperledger-fabric.readthedocs.io/en/latest/prereqs.html)
+- Follow the Fabric documentation for [downloading the Fabric samples and binaries](https://hyperledger-fabric.readthedocs.io/en/latest/install.html). You can skip the docker image downloads by using `curl -sSL https://bit.ly/2ysbOFE | bash -s -- -d`
+
+# Instructions for starting network
+
+Open terminal windows for 3 ordering nodes, 4 peer nodes, and 4 peer admins as seen in the following terminal setup. The first two peers and peer admins belong to Org1, the latter two peer and peer admins belong to Org2.
+Note, you can start with two ordering nodes and a single Org1 peer node and single Org1 peer admin terminal if you would like to keep things even more minimal (two ordering nodes are required to achieve consensus (2 of 3), while a single peer from Org1 can be utilized since the endorsement policy is set as any single organization).
+![Terminal setup](terminal_setup.png)
+
+The following instructions will have you run simple bash scripts that set environment variable overrides for a component and then runs the component.
+The scripts contain only simple single-line commands so that they are easy to read and understand.
+If you have trouble running bash scripts in your environment, you can just as easily copy and paste the individual commands from the script files instead of running the script files.
+
+- cd to the `test-network-nano-bash` directory in each terminal window
+- In the first orderer terminal, run `./generate_artifacts.sh` to generate crypto material (calls cryptogen) and system and application channel genesis block and configuration transactions (calls configtxgen). The artifacts will be created in the `crypto-config` and `channel-artifacts` directories.
+- In the three orderer terminals, run `./orderer1.sh`, `./orderer2.sh`, `./orderer3.sh` respectively
+- In the four peer terminals, run `./peer1.sh`, `./peer2.sh`, `./peer3.sh`, `./peer4.sh` respectively
+- Note that each orderer and peer write their data (including their ledgers) to their own subdirectory under the `data` directory
+- In the four peer admin terminals, run `. peer1admin.sh`, `. peer2admin.sh`, `. peer3admin.sh`, `. peer4admin.sh` respectively
```

Comment:
FYI as I run zsh as the primary shell the `. ` approach doesn't work, so I just pull out the various parts and do them separately

Reply:
Running `source peer1admin.sh` should work shouldn't it?

Reply:
works, done.